COMP36212/course_info.tex

```diff
@@ -20,25 +18,16 @@ \section*{Aims}
 \section*{Syllabus}
 
 \begin{mymulticols}
-  \begin{itemize}
-  \item PART 1: OPTIMISATION AND NATURE-INSPIRED ALGORITHMS (8 hours)
     \begin{itemize}
-      \item Introduction to optimisation algorithms (2 hours)
-	applications for optimisation algorithms
-	local and global optimisation
-	methods based on derivatives
-	1.4 - direct search methods
-      \item Stochastic optimisation (2 hours)
-	grid search, random searches, multistart
-	Simulated annealing
-	particle swarm
-      \item Evolutionary algorithms (4 hours)
-	3.1 - basics of genetics and evolution
-	3.2 - evolutionary programming
-	3.3 - genetic algorithms
-	3.4 - evolution strategies
-	3.5 - genetic programming
-        \end{itemize}
+    \item PART 1: NUMERICAL STABILITY AND ACCURACY OF COMPUTATIONS (10 hours)
```

Comment:
Where did you get this list from? I usually copy them from some place like https://studentnet.cs.manchester.ac.uk/ugt/COMP36212/syllabus/

Did you do something similar?